src/ocf_cache.c

```diff
@@ -20,7 +20,17 @@ ocf_volume_t ocf_cache_get_volume(ocf_cache_t cache)
 
 int ocf_cache_set_name(ocf_cache_t cache, const char *src, size_t src_size)
 {
+	ocf_cache_t tmp_cache;
+	int result;
 	OCF_CHECK_NULL(cache);
+
+	result = ocf_mngt_cache_get_by_name(ocf_cache_get_ctx(cache), src,
```

Comment:
This function was meant to be simple setter. It would be better to have this check in _ocf_mngt_cache_start() before this function is called.

Reply:
Since it is a part of public API, shouldn't we try to prevent setting invalid cache name?

Reply:
Oops, it looks like a oversight. This function definitely should not be public.